modules/ols-creating-the-credentials-secret-using-cli.adoc

```diff
@@ -32,8 +37,20 @@ stringData:
 ----
 <1> The `apitoken` is not `base64` encoded.
 +
-.Example credential secret for {azure-official} {openai}
+.Example credential secret for {watsonx}
+[source,yaml, subs="attributes,verbatim"]
```

Comment:
No white space needed between options:
```suggestion
[source,yaml,subs="attributes,verbatim"]
```
Just a nit since it still works. But, there are a lot of these in the related files (outside of the scope of the PR) if you felt like cleaning them up.

Reply:
done